spec/index.bs

```diff
@@ -1335,9 +1363,16 @@ dictionary GPUDeviceDescriptor : GPUObjectDescriptorBase {
         The set of {{GPUFeatureName}} values in this sequence defines the exact set of
         [=features=] that must be enabled on the device.
 
+        Issue: This needs to be a sequence of strings, so unknown feature names
+        don't cause IDL exceptions (so they have the same runtime behavior as known
+        but unsupported feature names).
```

Comment:
I realized there is actually no problem here. requestDevice is an async function and it already is supposed to `reject` in the case of any item not supported by the adapter. So it doesn't matter whether it rejects due to IDL rules or rejects due to WebGPU rules (we could even make it the same exception type if we wanted).